doc/neomake.txt

```diff
@@ -56,7 +56,7 @@ fantastic vim plugins such as syntastic and dispatch.
 <
                                                              *:NeomakeCancelJob*
 :NeomakeCancelJob *{job_id}*
```

Comment:
This should not have `*` around it, resulting in a warning:

> Vim(helptags):E154: Duplicate tag "job_id" in file /home/user/.vim/neobundles/.neobundle/doc/neomake.txt

Same below.. just use `{jobid}` here (or  `{job_id}`, but it's not highlighted then correctly, which is a Vim issue though).

`*` is used incorrectly also below, but let's keep that for another PR (e.g. with `*filemode*`).
`*tag*` is meant to create a unique tag in the help files.